packages/flatpickr/src/input.css

```diff
@@ -0,0 +1,898 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.flatpickr-calendar {
+  background: transparent;
+  opacity: 0;
+  display: none;
+  text-align: center;
+  visibility: hidden;
+  padding: 0;
+  -webkit-animation: none;
+  animation: none;
+  direction: ltr;
+  border: 0;
+  font-size: 14px;
+  line-height: 24px;
+  position: absolute;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  -ms-touch-action: manipulation;
+  touch-action: manipulation;
+  border: 2px solid;
+  border-radius: 0px 0px 4px 4px;
+  @apply px-2 relative !top-[-2px] !left-0 py-3 bg-white border-primary text-black text-base;
+}
+
+.flatpickr-calendar.open,
+.flatpickr-calendar.inline {
+  opacity: 1;
+  max-height: 640px;
+  visibility: visible;
+}
+.flatpickr-calendar.open {
+  display: inline-block;
+  z-index: 99999;
+}
+.flatpickr-calendar.animate.open {
+  -webkit-animation: fpFadeInDown 300ms cubic-bezier(0.23, 1, 0.32, 1);
+  animation: fpFadeInDown 300ms cubic-bezier(0.23, 1, 0.32, 1);
+}
+.flatpickr-calendar.inline {
+  display: block;
+  position: relative;
+  top: 2px;
+}
+.flatpickr-calendar.static {
+  position: absolute;
+  top: calc(100% + 2px);
+}
+.flatpickr-calendar.static.open {
+  z-index: 999;
+  display: block;
+}
```

Comment:
Hm. In general I would have expected that we use more nesting etc. here.

Reply:
@mariohamann This file was largely copy-pasted from the default flatpickr theme. I only adjusted the classes I needed and left everything else as is. I thought it would be too much effort to refactor it since it is +900 lines. What do you propose here?